src/rpc/masternode.cpp

```diff
@@ -480,33 +480,35 @@ UniValue masternode(const JSONRPCRequest& request)
 
 UniValue masternodelist(const JSONRPCRequest& request)
 {
-    std::string strMode = "status";
+    std::string strMode = "json";
```

Comment:
Does this pose some risk for incompatibility?

Reply:
Well, yes and no :) `status` is still a valid mode, it's just not the default one anymore i.e. you can fix issues (if you have any) by specifying `status` explicitly. This PR follows the same idea as https://github.com/dashpay/dash/pull/1900 which is to move rpc output to json formatted data and _slowly_ deprecate custom strings we have now.